lib/cog/commands/help.ex

```diff
@@ -75,11 +59,81 @@ defmodule Cog.Commands.Help do
       {:ok, body} ->
         {:reply, req.reply_to, body, state}
       {:error, error} ->
-        {:error, req.reply_to, error, state}
+        {:error, req.reply_to, error_msg(error), state}
     end
   end
 
   def handle_message(req, state) do
     {:reply, req.reply_to, "usage", %{usage: @moduledoc}, state}
   end
+
+  # Note: We could do the command lookup first regardless of the presence of ':'
+  # This would eliminate the second case statement but would also mean that commands
+  # with the same name as bundles would take precedence.
+  defp lookup(name) do
+    # If the name contains a ':' then we just do a command lookup
```

Comment:
We need to tweak this to accommodate git-style commands: `help ec2:instances-ls` vs `help ec2:instances ls`. Easiest way to do that is run the args thru another parse cycle to resolve the actual command name.